apps/test/unit/lib/levelbuilder/lesson-editor/AddVocabularyDialogTest.jsx

```diff
@@ -101,4 +100,37 @@ describe('AddVocabularyDialog', () => {
     wrapper.update();
     expect(wrapper.find('h3').contains('There was an error'));
   });
+
+  it('renders default props', () => {
+    const wrapper = shallow(
+      <AddVocabularyDialog
+        {...defaultProps}
+        selectableLessons={[{id: 1, name: 'lesson1'}, {id: 2, name: 'lesson2'}]}
+      />
+    );
+    expect(wrapper.contains('Add Vocabulary')).to.be.true;
+    expect(wrapper.find('input').length).to.equal(2);
```

Comment:
"Add Vocabulary" does actually test business logic because the header is different in different cases. I see the point in removing line 112 so I'll go ahead and do this

Reply:
Ah, great, thank you for pointing that out! 👍

Reply:
> "Add Vocabulary" does actually test business logic because the header is different in different cases

It does! But I'd argue that means it should belong in a test that's specifically testing that the header is different in different cases, not be disguised as a test that claims to just be testing our ability to render